adversarial_workflow/utils/citations.py

```
@@ -0,0 +1,608 @@
+    return marked_document, results, blocked_tasks
+
+
+def print_verification_summary(results: list[URLResult]) -> None:
+    """Print a summary of verification results to stdout."""
+    available = sum(1 for r in results if r.status == URLStatus.AVAILABLE)
+    blocked = sum(1 for r in results if r.status == URLStatus.BLOCKED)
+    broken = sum(1 for r in results if r.status == URLStatus.BROKEN)
+    redirect = sum(1 for r in results if r.status == URLStatus.REDIRECT)
+
+    total = len(results)
+    print(f"\n📋 Citation Verification Summary")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)